### PR TITLE
Refactor/general ams

### PR DESCRIPTION
**Pull request by espressoGoddess**
### Files Changed:
-Sorry, not listing them all

### What Has Changed: 
- This changes all misnamed `class` to `className`
- This adds missing proptypes
- This styles various things from journal prompt, to the arrows being static
- This styles the logo to not be responsive
- This utilizes the `setUser` method when fetching data
- Replaces apostrophes with &apos;
- Change clickable `p` tags to be `button`s for semantics and a11y and update styles

### CODE REVIEW

File: client/src/Components/Activities/Activities.css [hunk 1]
```diff
@@ -12,6 +12,7 @@ html {
 .activity-card h2 {
   margin: 10px;
   font-size: xx-large;
+  max-width: 40vw;
```
Comment (chrissycooper) [inline client/src/Components/Activities/Activities.css:15]: nice

File: client/src/Components/Activities/Activities.js [hunk 2]
```diff
@@ -30,11 +30,11 @@ const Activities = ({ activities, addEntry, setFeeling }) => {
       <p className="activity-blurb">Here are a few activites that might meet you where you are...</p>
       <div className="activity-card">
         <div className="button-container">
-          {currentActivityIndex > 0 && <button className="activities-btns" onClick={prevActivity}><i class="fa-solid fa-arrow-left-long"></i></button>}
+          {currentActivityIndex > 0 && <button className="activities-btns" onClick={prevActivity}><i className="fa-solid fa-arrow-left-long"></i></button>}
```
Comment (jfogiato) [inline client/src/Components/Activities/Activities.js:33]: TY for taking the time to swap these out

File: client/src/Components/Activities/Activities.js [hunk 3]
```diff
@@ -30,11 +30,11 @@ const Activities = ({ activities, addEntry, setFeeling }) => {
...
         </div>
         <h2>{activities[currentActivityIndex]?.activity}</h2>
         <div className="button-container">
-          {currentActivityIndex < activities.length - 1 && <button className="activities-btns" onClick={nextActivity}><i class="fa-solid fa-arrow-right-long"></i></button>}
+          {currentActivityIndex < activities.length - 1 && <button className="activities-btns" onClick={nextActivity}><i className="fa-solid fa-arrow-right-long"></i></button>}
```
Comment (chrissycooper) [inline client/src/Components/Activities/Activities.js:37]: thank you!!! Sorry that slipped my mind

File: client/src/Components/App/App.js [hunk 4]
```diff
@@ -33,7 +33,10 @@ const App = () => {
       activity: activity,
       journal_entry: journal
     };
-    fetchCall(newEntry).then(updateLogs);
+    fetchCall(newEntry).then(() => {
+      updateLogs();
+      setUser({id: 1, name: 'KB'});
+    });
```
Comment (jfogiato) [inline client/src/Components/App/App.js]: Thank you for updating this!! Also allows for a fairly quick plug and play on the front end if/when we add user authentication

File: client/src/Components/Home/Home.css [hunk 5]
```diff
@@ -12,13 +12,13 @@
   align-items: center;
   text-align: center;
   justify-content: center;
-  border: 1px solid white;
+  border: 2px solid white;
```
Comment (chrissycooper) [inline client/src/Components/Home/Home.css:15]: yes! thank you, this is something I was thinking about as a second order adjustment.

File: client/src/Components/Home/Home.js [hunk 6]
```diff
@@ -8,16 +8,16 @@ const Home = () => {
   return (
     <section className='home'>
       <h1 className='logo-home'>BEAM</h1>
-      <p className='uni-btn' onClick={() => setBlurb(!blurb)}>What is Beam?</p>
+      <button className='uni-btn' onClick={() => setBlurb(!blurb)}>What is Beam?</button>
```
Comment (jfogiato) [inline client/src/Components/Home/Home.js:11]: Thank you for updating this to a button for MAX accessibility

File: client/src/Components/JournalPrompt/JournalPrompt.js [hunk 7]
```diff
@@ -23,8 +23,8 @@ const JournalPrompt = ( {updateJournal, feeling} ) => {
       <form>
         <h2 className="prompt">Why are you feeling this way?</h2>
         <textarea required id="journal-entry" onChange={(event) => setNewEntry(event.target.value)}></textarea>
-        <p className="uni-btn" onClick={handleClick}><i class="fa-solid fa-arrow-right-long"></i></p>
-        {error ? <p style={{textAlign: 'center'}}> Please tell us a bit more about how you're feeling today - you can always edit it later. 🤍 </p> : null}
+        <button className="uni-btn" onClick={handleClick}><i className="fa-solid fa-arrow-right-long"></i></button>
+        {error ? <p style={{textAlign: 'center'}}> Please tell us a bit more about how you&apos;re feeling today - you can always edit it later. 🤍 </p> : null}
       </form>
     </div>
```
Comment (chrissycooper) [inline client/src/Components/JournalPrompt/JournalPrompt.js:29]: Amazing, love it, thanks again

File: client/src/Components/Home/Home.js [hunk 8]
```diff
@@ -8,16 +8,16 @@ const Home = () => {
...
-            <h3 className='blurb'>We created a flow intended to help you work through any kind of feeling or emotion you might be experiencing. You'll be asked to choose a feeling and reflect on it. Add anything about your experience that might help you avoid or recreate the feeling in the future (depending on the feeling). After your reflection, you'll see an offering of a few activities that might help. <br/><br/>Your entries will be available to you in the Feelings Archive for future reflection. <br/> <br/>We hope this process will help reveal useful patterns over time.</h3> 
+            <h3 className='blurb'>We created a flow intended to help you work through any kind of feeling or emotion you might be experiencing. You&apos;ll be asked to choose a feeling and reflect on it. Add anything about your experience that might help you avoid or recreate the feeling in the future (depending on the feeling). After your reflection, you&apos;ll see an offering of a few activities that might help. <br/><br/>Your entries will be available to you in the Feelings Archive for future reflection. <br/> <br/>We hope this process will help reveal useful patterns over time.</h3>
```
Comment (jfogiato) [inline client/src/Components/Home/Home.js:16]: ESLint can pound salt now

**Review by chrissycooper — APPROVED** [review]

**Review by jfogiato — APPROVED** [review]
Thanks for getting into the weeds on this one, Amber!!

File: client/src/Components/JournalEntry/JournalEntry.js [hunk 9]
```diff
@@ -38,7 +38,7 @@ const JournalEntry = ({ journal, entryID, updateLogs }) => {
 
   const showEntry = <p className='journal-text'>{entry}</p>
   const entryButtons = <div>
-                          <Link to={'/how-you-felt'} className="uni-btn"><i class="fa-solid fa-arrow-left-long"></i></Link>
+                          <Link to={'/how-you-felt'} className="uni-btn"><i className="fa-solid fa-arrow-left-long"></i></Link>
```
Comment (KatherineBlaine) [inline client/src/Components/JournalEntry/JournalEntry.js:41]: THANK YOU for going through all of these! You rock!

**Review by KatherineBlaine — APPROVED** [review]
Thanks so much for all this cleanup Amber. Super helpful, you have a great attention to detail!!!

**Pull request merged**